src/application/cli.py

```
@@ -0,0 +1,205 @@
+from ..infrastructure.container import get_container
+from ..infrastructure.logging import setup_logging
+from ..infrastructure.cache import MemoryCache
+from ..infrastructure.storage import CSVStorage
+from ..infrastructure.metrics import PrometheusMetrics, MetricsReporter
+from ..plugins import get_registry
+from ..domain.models import ParameterType
+from .downloader import AirQualityDownloader
+from .job_manager import InMemoryJobManager
+
+
+async def main():
```

Ruff (0.12.2)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)